drivers/ipvlan/ipvlan_test.go

```diff
@@ -0,0 +1,59 @@
+package ipvlan
+
+import (
+	"testing"
+
+	"github.com/docker/libnetwork/driverapi"
```

Comment:
You need to add this import for the CI to pass:
`_ "github.com/docker/libnetwork/testutils"`